src/store/index.js

```diff
@@ -5,7 +5,7 @@ import rootReducer from '../ducks/reducer';
 
 const createStoreWithMiddleware = applyMiddleware(
   CreateJumpstateMiddleware(),
-  createLogger(),
+  // createLogger(),
```

Comment:
Reminder to future devs: re-enable this line of code to see `{prev state, action, next state}` Redux debug in the web console. It gets hella noisy but it's useful.